solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -103,6 +103,8 @@ public class SolrConfig implements MapSerializable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   public static final String DEFAULT_CONF_FILE = "solrconfig.xml";
+  public static final String MIN_PREFIX_LENGTH = "minPrefixLength";
```

Comment:
from a naming standpoint, this is very ambiguous.  Prefix of what?  And should this be here at all vs. a request parameter, interpreted by some (not all) query parsers?

Reply:
Does the "Term" in that add anything?  I suggest removing that component.  Not a strong opinion though.